assets/blocks/course-outline/outline-block/outline-placeholder.js

```diff
@@ -49,79 +49,91 @@ const EditPlaceholder = ( { addBlocks, openTailoredModal = noop } ) => {
 
 	return (
 		<Placeholder
-			className="wp-block-sensei-lms-course-outline__placeholder"
+			className="outline-placeholder"
```

Comment:
In my original PR, I had intentionally chosen this naming to match what our other blocks do and to follow Sensei's naming conventions. Given this, I think we should revert back to the previous class names.

Reply:
Fixed [Prefix classes with block name](https://github.com/Automattic/sensei/pull/7018/commits/a2326ae5d6d4e38cc87ae7257dfb078e28e4859f)